src/test/java/tests/uiCheckTests/CreateList.java

```diff
@@ -0,0 +1,56 @@
+package tests.uiCheckTests;
+
+import api.clients.ApiBoardClient;
+import api.clients.ApiListClient;
+import api.pojo.requests.BoardBuilder;
+import api.pojo.requests.ListBuilder;
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.Selenide;
+import com.codeborne.selenide.impl.CollectionElement;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+import tests.TestInit;
+import ui.fragments.AllBoardsFragment;
+import ui.fragments.BoardWorkSpace;
+import ui.pages.LoginPage;
+import utils.ElementUtil;
+
+import java.util.List;
+
+public class CreateList extends TestInit {
+
+    private SoftAssert softAssert = new SoftAssert();
+    private BoardBuilder boardBody = BoardBuilder.builder().build();
+    private ListBuilder listBuilder = ListBuilder.builder().build();
+    private ApiListClient apiListClient = new ApiListClient(BASE_URL);
+    private ApiBoardClient apiBoardClient = new ApiBoardClient(BASE_URL);
+    private AllBoardsFragment allBoardsFragment = new AllBoardsFragment();
+    private BoardWorkSpace boardWorkSpace = new BoardWorkSpace();
+    private String boardId;
+
+
+    @BeforeMethod
+    public void setUp() {
+        boardId = apiBoardClient.createNewBoard(boardBody, 200).getId();
+    }
+
+
+    @Test
+    public void checkCreateList() {
+        String listName;
+        listName = apiListClient.createNewList(listBuilder, boardId, 200).getName();
+        String boardBodyName = boardBody.getName();
+        Selenide.refresh();
```

Comment:
Separate this with an empty line
```suggestion
        String boardBodyName = boardBody.getName();
        
        Selenide.refresh();
```